crates/rethnet_eth/src/remote/eth.rs

```diff
@@ -88,7 +88,7 @@ where
     Ok(u64::from_str_radix(&s[2..], 16).expect("failed to parse u64"))
 }
 
-/// log object used in TransactionReceipt
+/// log object used in `TransactionReceipt`
```

Comment:
This one should have the `[]` brackets, since it's actually a code object (right?)